engine/src/workers/observability/ui/styles.css

```
@@ -0,0 +1,773 @@
+  --obs-text-ghost: var(--color-ink-ghost, #a3a09c);
+  --obs-accent: var(--color-accent, #b8420f);
+  --obs-accent-muted: var(--color-accent-muted, rgba(184, 66, 15, 0.1));
+  --obs-accent-border: var(--color-accent-border, rgba(184, 66, 15, 0.35));
+  --obs-ok: var(--color-ok, #356f3d);
+  --obs-ok-muted: var(--color-ok-muted, rgba(53, 111, 61, 0.12));
+  --obs-warn: var(--color-warn, #a87a00);
+  --obs-warn-muted: var(--color-warn-muted, rgba(168, 122, 0, 0.12));
+  --obs-alert: var(--color-alert, #ff0026);
+  --obs-alert-muted: var(--color-alert-muted, rgba(255, 0, 38, 0.08));
+  --obs-mono: var(--font-mono, "Geist Mono", ui-monospace, SFMono-Regular, Menlo, Consolas, monospace);
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)